### PR TITLE
initial

### PR DESCRIPTION
**Pull request by tsylbarnes**
Initial files for the emtr-tool, both code and docs for the shinylive app

### CODE REVIEW

File: app/R/hard_load.R [hunk 1]
```diff
@@ -0,0 +1,41 @@
+get_parameters = function() {
+  params = list(modelyear = 2024, ACC_LevyRate = 0.0153, ACC_MaxLeviableIncome = 139384, 
+                Accommodation_AbatementRate = 0.25, Accommodation_BaseRateThreshold_Mortgage = 0.3, 
+                Accommodation_BaseRateThreshold_Rent = 0.25, Accommodation_MaxRate_CoupleDeps_Single2_Deps_Area1 = 0,
```
Comment (MeghanStephensNZ) [inline app/R/hard_load.R]: Most of the Accommodation Supplement parameters are incorrectly 0, which I think is an error you might have inherited from somewhere else.

File: app/ui.R [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      ),
+          
+      h2(" ") # Add padding to bottom of page
+    )
+  )
+  ),
+  # Very basic overview of the tool
+  tabPanel("About", 
+           mainPanel(h2("About the Tool"),
+                     p("This tool is designed to make support policy analysts in understanding the EMTR profiles faced by a range of different scenario families in order to help inform the design of Tax and Welfare policy. "),
```
Comment (Luke-Symes-Tsy) [inline app/ui.R]: Typo:
```suggestion
                     p("This tool is designed to support policy analysts in understanding the EMTR profiles faced by a range of different scenario families in order to help inform the design of Tax and Welfare policy. "),
```

File: app/R/app_utilities.R [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+  X <- rbind(With_IWTC, Without_IWTC)
+  setorderv(X, "hours1")
+  
+  X[, EMTR := 1 - 1L*(shift(Net_Income,1L,type="lead")-Net_Income)]
+  X[, EMTR := zoo::na.locf(EMTR)]
+  
+  return(X)
+}
+
+# plotly - income composition
```
Comment (Luke-Symes-Tsy) [inline app/R/app_utilities.R]: I don't think you need this function any more? It seems like amounts_net_plot covers it now.

File: app/R/emtr.R [hunk 4]
```diff
@@ -0,0 +1,572 @@
+parameters_from_file <- function(parameters_file) {
```
Comment (Luke-Symes-Tsy) [inline app/R/emtr.R:1]: I don't think you use this function anywhere, you could change it back to the original here.

**Review by Luke-Symes-Tsy — APPROVED** [review]
Great work Liam, looking good, just a few suggestions.

**Pull request merged**